Sources/StreamChatUI/Appearance.swift

```diff
@@ -18,3 +18,40 @@ public struct Appearance {
 public extension Appearance {
     static var `default`: Appearance = .init()
 }
+
+// MARK: UIConfig deprecation
+
+@available(*, deprecated, message: "UIConfig has split into Appearance and Components")
+public typealias UIConfig = Appearance
+
+@available(*, deprecated, message: "UIConfig has split into Appearance and Components")
+public extension UIConfig {
+    @available(*, deprecated, renamed: "Fonts")
+    typealias Font = Fonts
+    
+    @available(*, deprecated, renamed: "fonts")
+    var font: Font {
+        get { fonts }
+        set { fonts = newValue }
+    }
+}
+
+@available(
+    *,
+    deprecated,
+    message: "UIConfigProvider has split into AppearanceProvider and ComponentsProvider. Use ThemeProvider if you need both."
+)
+public typealias UIConfigProvider = ThemeProvider
+
+@available(
+    *,
+    deprecated,
+    message: "UIConfigProvider has split into AppearanceProvider and ComponentsProvider. Use ThemeProvider if you need both."
+)
+public extension UIConfigProvider {
+    @available(*, deprecated, message: "uiConfig has split into appearance and components")
+    var uiConfig: UIConfig {
+        get { appearance }
+        set { appearance = newValue }
+    }
+}
```

Comment:
Can you please create a `Deprecations.swift` file and move it there? The same way we have it for LLC